### ISSUE TITLE
Make colour palettes exportable to design software

### ISSUE THREAD

**Issue by pauloneillft** [Financial-Times/o-colors#47]
http://www.colourlovers.com/web/blog/2007/11/08/color-palettes-in-adobe-swatch-exchange-ase

Adobe has its ASE (Adobe Swatch Exchange) format which is interoperable between all Adobe software. Maybe an open format would be better but Adobe fits for now.

This tool will be very helpful to product designers down the line who can install the swatch file in their PS or AI or FW to help create FT designs.

**Comment by kaelig** [comment]
Ping @simonwilliamsFT: let us know if you're making Sketch / Adobe swatch files for Next, it would be interesting to have a similar thing for o-colors

**Comment by pauloneillft** [comment]
@kaelig just discussed this again with @SarahHornsey but in the context of Sketch. Does Sketch make it any easier...

**Comment by simonwilliamsFT** [comment]
no
On 1 Jul 2015 10:38, "pauloneillft" <notifications@github.com> wrote:

> @kaelig <https://github.com/kaelig> just discussed this again with
> @SarahHornsey <https://github.com/SarahHornsey> but in the context of
> Sketch. Does Sketch make it any easier...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Financial-Times/o-colors/issues/47#issuecomment-117563915>
> .
>

-- 

------------------------------
This email was sent by a company owned by Pearson plc, registered office at 
80 Strand, London WC2R 0RL.  Registered in England and Wales with company 
number 53723.

**Comment by kaelig** [comment]
I can't find any resources on that. If you stumbled upon a plugin for named-colours, it'd be great to have a look.

**Comment by pauloneillft** [comment]
From my quick searches I couldn't find one. Perhaps one of our clever FT devs could create one ;)

**Comment by pauloneillft** [comment]
Salesforce LSG exporting ASE and CLR formats
http://www.lightningdesignsystem.com/resources/downloads#role=regular&status=all

**Comment by kaelig** [comment]
I've asked @jina, designer at Salesforce, since I'm unsure how they export to these formats. It might be a manual process.

**Comment by onishiweb** [comment]
Is there any update on this? Would people still find it useful?

**Comment by SarahHornsey** [comment]
Yup :-)

**Comment by AlbertoElias** [comment]
Yeah, this would be nice

**Comment by onishiweb** [comment]
@olipowell do you think you could look at how this could be done in Sketch and any other software you're using and see how easy it would be to maintain it. Then we can see how best to host it here too. Cheers.

**Comment by kaelig** [comment]
Alternatively you could use a service like https://brand.ai/

**Comment by olipowell** [comment]
Yup, already on it! I'm just working out some teething problems before we start using this beyond the team, but I have a library of components up and running in Sketch. I'll put together a smaller library of say o-buttons that we can start experimenting with on different platforms.
